.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
.vscode can be ignored imo

Reply:
Since I was planning on making the out of the box experience as good as possible. Assuming you'll stick with VSCode being the recommend editor, we should probably extend that file in order to facilitate that. 

Luckily, CMake Tools handles most of the things so this may not be as required.